streaming/src/main/scala/org/apache/spark/streaming/Checkpoint.scala

```diff
@@ -41,6 +41,13 @@ class Checkpoint(ssc: StreamingContext, val checkpointTime: Time)
   val checkpointDuration = ssc.checkpointDuration
   val pendingTimes = ssc.scheduler.getPendingTimes().toArray
   val sparkConfPairs = ssc.conf.getAll
+  @transient val recoverableAccuNameToId = ssc.recoverableAccuNameToId
+
+  // initialize from ssc.context after SPARK-13051
+  val trackedAccs: Array[AccumulableCheckpoint[_, _]] = Accumulators.originals.filter(ele =>
```

Comment:
I am wondering if we really need the `AccumulableCheckpoint` here, can it be type of `Array[Accumulable[_, _]]`?